### DISCUSSION TITLE
Debugging single file used as a task in workflow

### DISCUSSION THREAD

**Discussion by Marco31Bike** [YetiForceCompany/YetiForceCRM#16757] [How to]
Guys, how to throw to file debug of VTEmailTask.php which is used in workflow tasks?
var_dump returning error
What would be recommended? Or, What is the way of debugging workflows as a whole? (including manual trigger workflows)

thanks for help

**Comment by mariuszkrzaczkowski** [comment]
Depending on the problem you have, you can use the debugging mechanism.

**Reply by s2b-git** [reply]
If I want to go deeper and see all stuff executed (or failed) by each var or func. I guess built-in debugging dont do that?

**Reply by Marco31Bike** [reply]
> If I want to go deeper and see all stuff executed (or failed) by each var or func. I guess built-in debugging dont do that?

That is the reason I asked for recommended method to see everything what happening when certain task is executed :-)

Thanks
Marco

**Comment by vovpff** [comment]
php put_file_contents
